tests/benchmarks/multicore/idle_ppr/src/main.c

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#include <zephyr/kernel.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(idle_ppr, LOG_LEVEL_INF);
+
+/* Variables used to make CPU active for ~1 second */
+static struct k_timer my_timer;
+static bool timer_expired;
+
+void my_timer_handler(struct k_timer *dummy)
+{
+	timer_expired = true;
+}
+
+int main(void)
+{
+	int counter = 0;
+
+	LOG_INF("Multicore idle_ppr test on %s", CONFIG_BOARD_TARGET);
+	LOG_INF("Main sleeps for %d ms", CONFIG_TEST_SLEEP_DURATION_MS);
+
+	k_timer_init(&my_timer, my_timer_handler, NULL);
+
+	/* Run test forever */
+	while (1) {
+		timer_expired = false;
+
+		/* start a one-shot timer that expires after 1 second */
+		k_timer_start(&my_timer, K_MSEC(1000), K_NO_WAIT);
+
+		/* Keep CPU active for ~ 1 second */
+		while (!timer_expired) {
+			k_busy_wait(10000);
+			k_yield();
+		}
+
+		LOG_INF("Run %d", counter);
+		counter++;
+
+		/* Sleep / enter low power state */
+		k_msleep(CONFIG_TEST_SLEEP_DURATION_MS);
+	}
+
+	return 0;
+}
+
+#if defined(CONFIG_BOARD_NRF54H20DK_NRF54H20_CPUPPR)
```

Comment:
Stolen from the Tarpan Team. I have no idea what I'm doing.

Reply:
I did not check the code, just cmake/kconfig... Move this to a separate file, then only build it if `CONFIG_SOC_NRF54H20_CPUPPR` is defined

Reply:
I just add this. It looks like Power Management is not implemented for PPR core.
I suspect, this is not enough to make this test passing.

PR changed to Draft.